src/DotNetLightning.Core/Chain/KeysInterface.fs

```diff
@@ -60,7 +60,7 @@ type IKeysRepository =
     /// Must add signature to the PSBT *And* return the signature
     abstract member GetSignatureFor: psbt: PSBT * pubKey: PubKey -> TransactionSignature * PSBT
     /// Must add signature to the PSBT *And* return the signature
-    abstract member GenerateKeyFromBasepointAndSign: psbt: PSBT * pubkey: PubKey * basepoint: PubKey -> TransactionSignature * PSBT
+    abstract member GenerateKeyFromBasepointAndSign: psbt: PSBT -> pubkey: PubKey -> perCommitmentPoint: PerCommitmentPoint -> TransactionSignature * PSBT
```

Comment:
"This makes the code simpler and easier to understand.", so this commit is just a refactoring? if so, it's better to use this word in the commit title

Reply:
I've the changed the commit title to use the word "refactor".